libs/wingsdk/src/target-sim/topic.inflight.ts

```diff
@@ -50,7 +50,7 @@ export class Topic
         timestamp: new Date().toISOString(),
       });
 
-      await fnClient.invoke(message).catch((err) => {
+      void fnClient.invoke(message).catch((err) => {
```

Comment:
Add a comment explaining why we are not awaiting here.

I would even take it another step and call this with a nextTick callback. This will ensure that you first yield.

Reply:
Done